components/clarity-repl/src/repl/interpreter.rs

```diff
@@ -125,16 +125,6 @@ impl ClarityInterpreter {
     }
 
     pub fn run(
-        &mut self,
-        contract: &ClarityContract,
-        ast: Option<&ContractAST>,
-        cost_track: bool,
-        eval_hooks: Option<Vec<&mut dyn EvalHook>>,
-    ) -> Result<ExecutionResult, Vec<Diagnostic>> {
-        self.run_interpreter(contract, ast, cost_track, eval_hooks)
-    }
-
-    fn run_interpreter(
```

Comment:
`run` and `run_interpreter` remained from the time we had `run_wasm` that would run `clarity-wasm` and compare the result from `interpreter` and `wasm`